src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -556,6 +644,27 @@
 
     /******************************** Utility Functions *****************************/
 
+    /**
+     * @description: Call Apex method and return a Promise
+     * @param apexMethodName: Apex method name to be called
+     * @param params: Parameters for apex method, if any required
+     */
+    callApex: function(component, apexMethodName, params) {
+        let helper = this;
```

Comment:
Pass 'this' directly? Or keep explicitly naming it 'helper'? Not a critical decision